spec/rspec/core/configuration_spec.rb

```diff
@@ -2375,20 +2373,13 @@ def metadata_hash(*args)
       end
 
       it 'overrides existing definitions of the aliased method name without issueing warnings' do
-        config.expose_dsl_globally = true
-
         class << ExampleGroup
           def my_group_method; :original; end
         end
 
-        Module.class_exec do
-          def my_group_method; :original; end
-        end
-
         config.alias_example_group_to :my_group_method
 
         expect(ExampleGroup.my_group_method).to be < ExampleGroup
-        expect(Module.new.my_group_method).to be < ExampleGroup
```

Comment:
What was this changed?

Reply:
We do not monkey-patch `Module` anymore. Example group DSL is not exposed on it.

As far as I understand, in this example, we check if aliasing an example group to `my_group_method` override `ExampleGroup`'s and `Module`'s own definitions of an identically named method. This doesn't work for `Module` anymore. `Module.new.my_group_method` would return `:original`, and `< ExampleGroup` check would fail.

Reply:
Got it